From: Hiral Shah <hishah@cisco.com>
Date: Thu, 20 Feb 2014 16:13:48 -0800
Subject: fnic: HDS: VMware host not recovering path after switch reboot (san boot)
Patch-Mainline: Not yet
References: bnc#863365,FATE#315811

Printing the vifid and fc_map in flogi_reject function in fnic layer might
be invalid if CVL was received prior to receiving the reject packet from
switch. It is reported by Hitachi. Test drop with this change is verified
to be fixed by Hitachi

Signed-off-by: Hiral Shah <hishah@cisco.com>
Acked-by: Hannes Reinecke <hare@suse.de>
---
 drivers/scsi/fnic/fnic_fcs.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/scsi/fnic/fnic_fcs.c b/drivers/scsi/fnic/fnic_fcs.c
index f15553a..84f464e 100644
--- a/drivers/scsi/fnic/fnic_fcs.c
+++ b/drivers/scsi/fnic/fnic_fcs.c
@@ -297,11 +297,6 @@ static inline int is_fnic_fip_flogi_reject(struct fcoe_ctlr *fip,
 
 	if (desc->fip_dtype == FIP_DT_FLOGI) {
 
-		shost_printk(KERN_DEBUG, lport->host,
-			  " FIP TYPE FLOGI: fab name:%llx "
-			  "vfid:%d map:%x\n",
-			  fip->sel_fcf->fabric_name, fip->sel_fcf->vfid,
-			  fip->sel_fcf->fc_map);
 		if (dlen < sizeof(*els) + sizeof(*fh) + 1)
 			return 0;
 
-- 
1.7.12.4

